.github/workflows/staging-build-deploy.yml

```
@@ -0,0 +1,63 @@
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build all modules
+        run: |
+          ./gradlew clean build -p aics-admin -x test
+          ./gradlew clean build -p aics-api -x test
+          ./gradlew clean build -p aics-auth -x test
+
+      - name: Build Docker images
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:28: Double quote to prevent globbing and word splitting

(shellcheck)